test/config/singlepathtest.go

```diff
@@ -43,19 +38,13 @@ func (s *TestSuite) TestSinglePath(t *testing.T) {
 	devicePath := gnmi.GetTargetPathWithValue(simulator.Name(), tzPath, tzValue, proto.StringVal)
 
 	// Set a value using gNMI client
-	gnmi.SetGNMIValueOrFail(t, gnmiClient, devicePath, gnmi.NoPaths, gnmi.NoExtensions)
-
-	err := gnmi.WaitForConfigurationCompleteOrFail(t, configapi.ConfigurationID(simulator.Name()), time.Minute)
-	assert.NoError(t, err)
+	gnmi.SetGNMIValueOrFail(t, gnmiClient, devicePath, gnmi.NoPaths, gnmi.SyncExtension(t))
```

Comment:
this import in some of the tests called gnmi in some others gnmiutils. So I think we should use gnmiutils to be more clear.